libopenfpga/libfpgaaesencryption/test/AES_main.cpp

```diff
@@ -0,0 +1,211 @@
+#include "AESfunctions.h"
+#include <fstream> 
+#include <string>
+#include <sstream>
+#include <algorithm>
+#include<cctype>
+
+//String conversion function
+string convertToString(char* a, int size)
```

Comment:
I would suggest to use the built-in constructor of the std::string. 

https://www.geeksforgeeks.org/convert-character-array-to-string-in-c/